app/views/step_by_step_pages/_change_notes.html.erb

```diff
@@ -13,6 +13,7 @@
   <div class="govuk-grid-column-two-thirds" id="change-note">
     <%= render "govuk_publishing_components/components/radio", {
       heading: "Notify users about this change?",
+      heading_size: 's',
```

Comment:
Nitpick: these could be double quotes like the rest of the config.

Reply:
If only the linter would work on template files too..
Thanks for the review @ChrisBAshton, fixed both mistakes.